web/src/components/RasaWidget.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { ReactElement, useEffect } from 'react'
+import { useTheme } from 'styled-components'
+
+const RasaWidget = (): ReactElement => {
+  const theme = useTheme()
+
+  useEffect(() => {
+    const script = document.createElement('script')
+
+    script.src = 'https://unpkg.com/@rasahq/rasa-chat'
+    script.async = false
+
+    document.body.appendChild(script)
+
+    return () => {
+      document.body.removeChild(script)
+      const button = document.getElementById('rasa-chat-widget-container')
```

Comment:
Shouldn't this be (matching the id below):
```suggestion
      const button = document.getElementById('rasa-chat-widget')
```

Reply:
It won't work as the script only takes the element with the id 'rasa-chat-widget', but does not render it directly. Have a look at the DOM and you will see, there is no element with this id.